projects/dsp-ui/src/lib/action/components/login-form/login-form.component.ts

```diff
@@ -0,0 +1,200 @@
+import { Component, EventEmitter, Inject, Input, OnInit, Output } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+import { ApiResponseData, ApiResponseError, KnoraApiConnection, LoginResponse, LogoutResponse } from '@dasch-swiss/dsp-js';
+import { DspApiConnectionToken, Session, SessionService } from '../../../core';
+
+@Component({
+  selector: 'dsp-login-form',
+  templateUrl: './login-form.component.html',
+  styleUrls: ['./login-form.component.scss']
+})
+export class LoginFormComponent implements OnInit {
+
+    /**
+     * Navigate to the defined url (or path) after successful login
+     *
+     * @param navigate
+     */
+    @Input() navigate?: string;
+
+    /**
+     * Set your theme color here,
+     * it will be used in the progress-indicator and the buttons
+     *
+     * @param color
+     */
+    @Input() color?: string;
+
+    /**
+     * Set whether or not you want icons to display in the input fields
+     *
+     * @param icons
+     */
+    @Input() icons?: boolean;
+
+    /**
+     * Emits true when the login process was successful and false in case of error
+     *
+     * @param loginSuccess
+     *
+     */
+    @Output() loginSuccess: EventEmitter<boolean> = new EventEmitter<boolean>();
+
+    /**
+     * Emits true when the logout process was successful and false in case of error
+     *
+     * @param logoutSuccess
+     *
+     */
+    @Output() logoutSuccess: EventEmitter<boolean> = new EventEmitter<boolean>();
+
+    // is there already a valid session?
+    session: Session;
+
+    // form
+    form: FormGroup;
+
+    // show progress indicator
+    loading = false;
+
+    // general error message
+    errorMessage: ApiResponseError;
+
+    // specific error messages
+    loginErrorUser = false;
+    loginErrorPw = false;
+    loginErrorServer = false;
+
+    // labels for the login form
+    // TODO: should be handled by translation service (i18n)
+    formLabel = {
+        title: 'Login here',
+        name: 'Username',
+        pw: 'Password',
+        submit: 'Login',
+        retry: 'Retry',
+        logout: 'LOGOUT',
+        remember: 'Remember me',
+        forgot_pw: 'Forgot password?',
+        error: {
+            failed: 'Password or username is wrong',
+            server: 'There\'s an error with the server connection. Try it again later or inform the Knora Team'
```

Comment:
@mdelez @kilchenmann small detail: the final dot is missing, and should we now use "Dasch Team" instead of "Knora Team"?

Reply:
Yes, it should be "DaSCH Team."